server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/service/EventDeadLettersService.java

```diff
@@ -38,6 +39,7 @@
     private final EventDeadLetters deadLetters;
 
     @Inject
+    @VisibleForTesting
```

Comment:
you don't need this annotation when the method is public, right?

Reply:
We do CF recuring comments from @mbaechler

Reply:
Yes @mbaechler asked me to add it. In fact I looked more into the matter and that annotation does nothing (if you take the public out, even with that your test class will not compile). It's just there to justify to other devs why we put for example here this constructor as public (for testing) because except for that, there is no need.